### PR TITLE
 [Task] Add batch edit based on folder ids

### LINKED ISSUE #494: [Task] Add endpoint to Batch-Edit whole folder
See #440

### PR DESCRIPTION
**Pull request by mattamon**
## Changes in this pull request
Resolves #494 

## Additional info
Adding new endpoint to batch edit data from elements based on folder id

### CODE REVIEW

File: src/Asset/Controller/PatchFolderController.php [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+     * @throws AccessDeniedException|ElementSavingFailedException
+     * @throws NotFoundException|UserNotFoundException|InvalidArgumentException
+     */
+    #[Route('/assets/folder', name: 'pimcore_studio_api_patch_asset_folder', methods: ['PATCH'])]
+    #[IsGranted(UserPermissions::ASSETS->value)]
+    #[Patch(
+        path: self::PREFIX . '/assets/folder',
+        operationId: 'asset_patch_folder_by_id',
+        description: 'asset_patch_folder_by_id_description',
+        summary: 'asset_patch_folder_by_id_summary',
```
Comment (lukmzig) [inline src/Asset/Controller/PatchFolderController.php:66]: Could we maybe add translations for the new controller action? :)

**Review by lukmzig — CHANGES_REQUESTED** [review]
Rest looks LGTM :seal:

File: src/Asset/Controller/PatchFolderController.php (hunk 1 above)
Comment (mattamon) [reply src/Asset/Controller/PatchFolderController.php:66]: haha yeah of course I forgot the translations. I will add them! :D

**Comment by mattamon** [comment]
@lukmzig sorry I forgot to add the filters so I had to refactor it again. 
I do however see a problem with the constants for the Generic Execution Engine. I do not like the current position, so I will move it to a more fitting place with a separate PR if that is okay.

**Review by lukmzig — APPROVED** [review]

**Pull request merged**